class.jetpack-jitm.php

```diff
@@ -232,7 +232,7 @@ static function get_messages( $message_path, $query ) {
 		), sprintf( '/sites/%d/jitm/%s', $site_id, $message_path ) );
 
 		// attempt to get from cache
-		$envelopes  = get_transient( 'jetpack_jitm_' . $path );
+		$envelopes  = get_transient( sha1( 'jetpack_jitm_' . $path ) );
```

Comment:
Instead of having the whole transient name going through `sha1`, I would suggest only hashing `$path`. This way, folks taking a look at their transient list can still figure out what the transient is used for, and what would happen if they were to delete it. It also makes it easier to search for a transient.

Here is an example:
https://github.com/Automattic/jetpack/blob/master/functions.opengraph.php#L301-L310

To ensure that your strings are never too long, you could also take this approach: `substr( md5( $path ), 0, 31 )`